api/controller/readcsv_controller.go

```diff
@@ -0,0 +1,57 @@
+package controller
+
+import (
+	model "api/models"
+	"encoding/csv"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/unrolled/render"
+)
+
+func ReadCSV(w http.ResponseWriter, r *http.Request) {
```

Comment:
Your handler name should reflect similarly to the route their handling, if you're getting contestants on the path `/api/contestants` then you would like to name this `GetContestants`. Similarly if your route is to get 1 contestant only `/api/contestants/{id}` then this could be named `GetContestant`